app/scripts/controller-init/confirmations/transaction-controller-init.ts

```diff
@@ -123,6 +123,7 @@ export const TransactionControllerInit: ControllerInitFunction<
         );
       },
     },
+    publicKeyEIP7702: process.env.EIP_7702_PUBLIC_KEY as Hex | undefined,
```

Comment:
Should this environment variable be added to `.metamaskrc.dist` with a comment on its usage?

Reply:
It absolutely should! I'll add that in a subsequent PR as it's just documentation to avoid delaying the cherry pick.